### PR TITLE
FI-1498-accept-headers

### PR DESCRIPTION
**Pull request by Jack-Fraser16**
Added accept header to `get` request made in test 5.2.05 to address discrepancy noted [here](https://github.com/onc-healthit/onc-certification-g10-test-kit/issues/47). This PR fixes GitHub Issue #47

### CODE REVIEW

File: lib/onc_certification_g10_test_kit/bulk_data_group_export.rb [hunk 1]
```diff
@@ -161,7 +161,7 @@ class BulkDataGroupExport < Inferno::TestGroup
         start = Time.now
 
         loop do
-          get(polling_url, headers: { authorization: "Bearer #{bearer_token}" })
+          get(polling_url, headers: { authorization: "Bearer #{bearer_token}", accept: 'application/fhir+json' })
```
Comment (yunwwang) [inline lib/onc_certification_g10_test_kit/bulk_data_group_export.rb:164]: It should be 'application/json'

"When requesting status, the client SHOULD use an Accept header indicating a content type of application/json."

**Review by yunwwang — APPROVED** [review]

File: spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb [hunk 2]
```diff
@@ -247,7 +247,9 @@ def run(runnable, inputs = {})
       expect(result.result_message).to eq('Complete Status response did not contain "transactionTime" as required')
     end
 
-    it 'passes when server returns "202 Accepted" and response body contains required attributes' do
+    it 'passes when server returns "202 
+    
+    ed" and response body contains required attributes' do
```
Comment (arscan) [inline spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb:252]:
```suggestion
    it 'passes when server returns "202 Accepted" and response body contains required attributes' do
```

Comment (arscan) [reply spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb:252]: Actually, why is this changed at all?  Just a rogue change?

File: spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb [hunk 3]
```diff
@@ -256,6 +258,16 @@ def run(runnable, inputs = {})
 
       expect(result.result).to eq('pass')
     end
+
+    it 'sends content type headers' do
+      get_stub = stub_request(:get, polling_url.to_s)
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .with(headers: { 'Accept' => 'application/json' })
```
Comment (arscan) [inline spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb:265]: Why not just put both in one hash?  I'm a bit concerned the second will clobber the first...  i did a quick scan of the documentation and didnt see anything on this.

File: spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb (hunk 2 above)
Comment (chris-at-work) [reply spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb]: Idk.  Pushed fix.

File: spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb (hunk 3 above)
Comment (chris-at-work) [reply spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb]: Yeah, probably safer all in one, pushed.

**Comment by Jammjammjamm** [comment]
Please add the ticket to the title before merging.

**Review by arscan — APPROVED** [review]
I did a quick search throughout our new bulk tests for v2.0 and didn't see anywhere else that this would be a problem.

**Pull request merged**